### PR TITLE
Project: Decode a Morse code message

### PR DESCRIPTION
**Pull request by mukhammadrizooff**
# In this project we implement:
- We create a method to decode a Morse code character, takes a string parameter, and return the corresponding character in uppercase (e.g. `decode_char(".-")` returns `"A"`).
- We create a method to decode an entire word in Morse code, take a string parameter, and return the string representation. Every character in a word was separated by a single space (e.g. `decode_word("-- -.--")` returns `"MY"`).
- We create a method to decode the entire message in Morse code, takes a string parameter, and return the string representation. Every word was separated by 3 spaces (e.g.
    ```
      decode("-- -.--   -. .- -- .")
    ```
  
  returns `"MY NAME"`).
- Our code is ready to use to decode the message from the old bottle:

```
      .-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...
```
      
      
  We use IRB for that :)

### CODE REVIEW

File: decode_morse.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+end
+
+def decode(param2)
+  param1 = param2
+  text = ''
+  param1.split('   ').each do |word|
+    text += decode_word(word)
+    text += ' '
+  end
+  puts text
```
Comment (rukundoeric) [inline decode_morse.rb]: - [x] <b>REQUIRED:</b> The `puts` on line 61 is messing up with the out put of the  function. checkout the screenshot below: 
scrnli_4_14_2022_6-54-06 PM
Kindly remove it. 

```ruby
def decode(param2)
  param1 = param2
  text = ''
  param1.split('   ').each do |word|
    text += decode_word(word)
    text += ' '
  end
  text
end

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @mukhammadrizooff, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: decode_morse.rb (hunk 1 above)
Comment (mukhammadrizooff) [reply decode_morse.rb]: We will implement soon as possible!

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**